src/authentication/Login/LoginRequest.tsx

```diff
@@ -0,0 +1,39 @@
+import storage from '../../app/localStorage';
+
+const LoginRequest = async (username:string, password:string) => {
+let response = {
+error: '',
+};
+if (username.trim().length !== 0 && password.trim().length !== 0) {
+    try {
+        const result = await fetch('openmrs/ws/rest/v1/session', {
+        headers: {
+            Authorization: 'Basic ' + btoa(username + ':' + password),
+        },
+        method: 'GET',
+        redirect: 'follow',
+        });
+        const [headers, body] = await Promise.all([
+        result.headers,
+        result.json(),
+        ]);
```

Comment:
I would recommend you extract this into an `authentication.resource.ts` file

Reply:
Resolved